src/components/MainComponent/FoodCards/FoodCard.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import "./FoodCards.scss";
+
+class FoodCard extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            inputValue: "0",
+            inputId: "",
+        }
+    }
+
+    handleChange = (event) => {
+        const inputValue = event.target.value;
+        const inputId = this.props.id;
+
+        this.setState({ inputValue, inputId });
+    };
+
+    handleKeyPress = (event) => {
```

Comment:
You can implement the same logic just define the input's type="number" and easily hide arrows via css